src/dnlib.netstandard.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <RootNamespace>dnlib</RootNamespace>
+        <AssemblyName>dnlib</AssemblyName>
+        <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+        <SignAssembly>true</SignAssembly>
+        <AssemblyOriginatorKeyFile>..\dnlib.snk</AssemblyOriginatorKeyFile>
+        <OutputPath>../$(Configuration)/bin</OutputPath>
```

Comment:
I'm pretty sure backslashes can still be used. MSBuild should convert them to forward slashes if needed